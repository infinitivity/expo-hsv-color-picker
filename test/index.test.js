import React from 'react';
import renderer from 'react-test-renderer';
import HsvColorPicker from '../src';

describe('<HsvColorPicker />', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<HsvColorPicker />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
