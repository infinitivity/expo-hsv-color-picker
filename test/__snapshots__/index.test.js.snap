// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HsvColorPicker /> renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
      },
      Object {},
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        },
        Object {},
        Object {
          "height": 224,
          "width": 224,
        },
      ]
    }
  >
    <ExponentLinearGradient
      accessible={true}
      colors={
        Array [
          4294967295,
          4294901760,
        ]
      }
      endPoint={
        Array [
          1,
          0.5,
        ]
      }
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      startPoint={
        Array [
          0,
          0.5,
        ]
      }
      style={
        Object {
          "borderRadius": 0,
        }
      }
    >
      <ExponentLinearGradient
        colors={
          Array [
            0,
            4278190080,
          ]
        }
      >
        <View
          style={
            Object {
              "height": 200,
              "width": 200,
            }
          }
        />
      </ExponentLinearGradient>
    </ExponentLinearGradient>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Array [
          Object {
            "borderColor": "#fff",
            "left": 0,
            "position": "absolute",
            "top": 0,
          },
          Object {
            "backgroundColor": "#ff0000",
            "borderRadius": 12,
            "borderWidth": 2.4,
            "height": 24,
            "transform": Array [
              Object {
                "translateX": 200,
              },
              Object {
                "translateY": 0,
              },
            ],
            "width": 24,
          },
        ]
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        },
        Object {},
        Object {
          "paddingBottom": 12,
          "paddingLeft": 6,
          "paddingRight": 6,
          "paddingTop": 12,
        },
      ]
    }
  >
    <ExponentLinearGradient
      accessible={true}
      colors={
        Array [
          4294901760,
          4294967040,
          4278255360,
          4278255615,
          4278190335,
          4294902015,
          4294901760,
        ]
      }
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "borderRadius": 0,
        }
      }
    >
      <View
        style={
          Object {
            "height": 200,
            "width": 12,
          }
        }
      />
    </ExponentLinearGradient>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Array [
          Object {
            "borderColor": "#fff",
            "position": "absolute",
            "top": 0,
          },
          Object {
            "backgroundColor": "#ff0000",
            "borderRadius": 12,
            "borderWidth": 2.4,
            "height": 24,
            "transform": Array [
              Object {
                "translateY": 0,
              },
            ],
            "width": 24,
          },
        ]
      }
    />
  </View>
</View>
`;
